internal/core/error.go

```diff
@@ -68,6 +69,11 @@ func (s *CliError) MarshalHuman() (string, error) {
 		sections = append(sections, str)
 	}
 
+	if sdkResponseError, isResponseError := s.Err.(*scw.ResponseError); isResponseError && s.Details == "" && s.Hint == "" {
```

Comment:
Are SDK error always wrapped in a CliError 🤔 ? If we explicitly catch an SDK error to put it inside a cli error this means we know what we were doing. Maybe this should be handled in the printer in case an SDK error bubble all the way up to the printer.